test/Microsoft.Sbom.Api.Tests/Converters/SbomToolManifestPathConverterTests.cs

```diff
@@ -37,85 +40,167 @@ public void Setup()
     }
 
     [TestMethod]
-    public void SbomToolManifestPathConverterTests_ValidPath_Succeeds()
+    public void SbomToolManifestPathConverterTests_ValidPath_Succeeds_Windows()
     {
+        var os = OSPlatform.Windows;
+        if (!isWindows)
+        {
+            // End the test for windows if the os is not windows
+            return;
```

Comment:
Actually, instead of `return;` on these can we do `Assert.Inconclusive()`? I get nervous when we should green checkmarks for tests that didn't actually run.